mathics/builtin/box/inout.py

```diff
@@ -1,9 +1,13 @@
 # -*- coding: utf-8 -*-
 
-from mathics.builtin.base import BoxExpression
+"""
+Low level notation structures. 
```

Comment:
I don't understand what this means. I would appreciate a long and as detailed as you can give. I don't mind care about English grammar, write in Spanish if that helps. What I am interested in is are the idea about what you think is or should be going on in this module.

Reply:
This module contains almost the same that we found in https://reference.wolfram.com/language/guide/LowLevelNotebookStructure.html. I didn´t go beyond on the description because what I have in mind is probably not accurate enough to be useful. What is true is that all these structures are "Builtin symbols" which are the heads of formatted (boxed) expressions.

Reply:
Ah. Thanks - I now understand.

Here is a rewording of the information there adjusted in a way that is more with the focus of something _implementing_ tor trying to understand and implementation as opposed to a user sees things at the "what does this do for me" level (used in Notebooks). 

_Formatting constructs are represented as a hierarchy of low-level symbolic "boxes"._ 

_The routines here assist in boxing at the bottom of the hierarchy. At the other end, the top level, we have a Notebook which is just a collection of Expressions usually contained in boxes._

Reply:
Given what i now understand is in `inout`,  I think that is not a great name.  Something like "base" or "core" would make more sense.